push/exec/table_exec.go

```diff
@@ -37,13 +38,14 @@ type TableExecutor struct {
 	fillTableID       uint64
 	delayer           interruptor.InterruptManager
 	transient         bool
+	ingestOnFirstMV   bool
```

Comment:
I don't think TableExecutor needs this flag - it's not used internally, it's just being set here in order to pass it somewhere else.

Reply:
Done, removed from table_exec